### DISCUSSION TITLE
Sigma Detections and Alertes view

### DISCUSSION THREAD

**Discussion by JulienDaSilvaUPPA** [Security-Onion-Solutions/securityonion#13319] [Unsupported Versions]
Hello,
I have a Standalone SO which works for Network detections. And now, I'm trying to test the new feature "Detection".
I have install an elastic Agent on a Debian12, I saw the agent in Elastic fleet (firewall config is ok), I applied an "agent policy"  containing "System integration" (which collecting /var/log/auth.log*)
In the "Detection view" I want to test a simple sigma rule "Privileged User Has Been Created"  which detection work like this : 
```logsource:
    product: linux
    definition: /var/log/secure on REHL systems or /var/log/auth.log on debian like Systems needs to be collected in order for this detection to work
detection:
    selection_new_user:
        - new user
    selection_uids_gids:
        - GID=0
        - UID=0
        - GID=10
        - GID=27
    condition: all of selection_*
falsepositives:
    - Administrative activity
level: high
```

I enabled the rule and so I created a new user with GID=0 on my Debian12, I can see the event in Dashbord or Hunt views but never in the Alerte view

Can you help me please
Julien

**Comment by defensivedepth** [comment]
Have you tested the Detection by clicking on the "Convert" button and then using the query to search for the log?

**Reply by JulienDaSilvaUPPA** [reply]
Hello and thank you for your answer but, where can I use this query ? it looks like this : 
```any where ""new user"" and (""GID=0"" or ""UID=0"" or ""GID=10"" or ""GID=27"")```

**Reply by defensivedepth** [reply]
There is a button that says "Test in Kibana", you can use that. 

But, I can already tell you that it looks like the field mapping for that rule needs to be updated. I have made a note of it to get it fixed.

**Comment by JulienDaSilvaUPPA** [comment]
Hello,
It doesn't work, there is a syntax error
![Screenshot_20240715_154856](https://github.com/user-attachments/assets/ac99ca1e-05de-4a62-a45c-c5c434acd4e3)
![Screenshot_20240715_155135](https://github.com/user-attachments/assets/47a0e44c-c065-4b12-9649-d01128b66b21)
...but when I manually search I find the log in Kibana: 
![Screenshot_20240715_161858](https://github.com/user-attachments/assets/cbee27d9-7fdf-4a80-966d-dbcdb024dcc2)
Moreover why does sigma rule use "GID=0"  (that we can find in auth.log file)  whereas "group.id" is used in Kibana

Thank you for your answers
Julien

**Reply by defensivedepth** [reply]
Again, these are mapping issues. We are looking into it.
